css/css-ui/appearance-button-drop-down-select-001.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: appearance: button on drop-down select</title>
+<link rel="help" href="https://drafts.csswg.org/css-ui-4/#appearance-switching">
+<meta name="assert" content="The button value works on a drop-down select.">
+<link rel="mismatch" href="drop-down-select-ref.html">
+<link rel="mismatch" href="drop-down-select-appearance-none-ref.html">
+<style>
+ select { -webkit-appearance: button; }
```

Comment:
I am of two minds about this. The spec does mandate that the -webkit- prefix be supported, so it's not wrong to have tests with it, but we should test the unprefixed name as well. One approach would be to duplicate all the tests, so that we have a prefixed and an unprefixed variant of each. We could also have a lot of tests covering one of the names and a few checking that the other name works as well, but in that case we usually go with the unprefixed name for most tests.

Either way, I'm OK with using testing the prefixed name, as long as we're not **only** testing the prefixed name.

Reply:
(this comment applies to all tests in this PR)

Reply:
A few tests test both (like JS tests where that is easy). If we want to duplicate the tests I think I would write a build script to generate them, though I wanted to keep things simple. :)

Reply:
I wrote the script in https://github.com/web-platform-tests/wpt/pull/14996 , when that is merged I can rebase the remaining PRs and fix as appropriate.

Reply:
Actually I fixed this PR without rebasing (I just ran the script without committing the script file itself here).